### DISCUSSION TITLE
Suggestion: Change email in "Contributing" and "Security" files since dane is no longer maintainer

### DISCUSSION THREAD

**Discussion by VasilisThePikachu** [pterodactyl/panel#4404] [Feature Requests]
Since Dane has stepped out from maintainer of pterodactyl it would be best to change danes email in [Contributing](https://github.com/pterodactyl/panel/blob/develop/CONTRIBUTING.md) and [Security](https://github.com/pterodactyl/panel/blob/develop/SECURITY.md) to Matthews or someone else in the ptero team.

I was not sure if i should make an issue for this so i made a discussion to be safe

**Comment by trenutoo** [comment]
This has been solved in https://github.com/pterodactyl/panel/commit/ba10646e8a3d61814d01f88bfc544a2cb8c07193
